ui.apps/src/main/content/jcr_root/apps/core/fd/components/form/tabsontop/v1/tabsontop/clientlibs/editor/js/tabs.js

```diff
@@ -0,0 +1,79 @@
+/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ ~ Copyright 2018 Adobe
+ ~
+ ~ Licensed under the Apache License, Version 2.0 (the "License");
+ ~ you may not use this file except in compliance with the License.
+ ~ You may obtain a copy of the License at
+ ~
+ ~     http://www.apache.org/licenses/LICENSE-2.0
+ ~
+ ~ Unless required by applicable law or agreed to in writing, software
+ ~ distributed under the License is distributed on an "AS IS" BASIS,
+ ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ ~ See the License for the specific language governing permissions and
+ ~ limitations under the License.
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/
+(function($, Coral) {
+    "use strict";
+
+    var dialogContentSelector = ".cmp-tabs__editor";
+    var childreneditorSelector = "[data-cmp-is='childrenEditor']";
+    var activeItemSelector = "[data-cmp-tabs-v1-dialog-edit-hook='activeItem']";
+    var activeSelectSelector = "[data-cmp-tabs-v1-dialog-edit-hook='activeSelect']";
+
+    $(document).on("dialog-loaded", function(e) {
```

Comment:
can we borrow this directly from sites library?

Reply:
@dmaurya929  can you check this?

Reply:
Okk will have a look.

Reply:
done